### DISCUSSION TITLE
Leverage

### DISCUSSION THREAD

**Discussion by SW4T400** [edtechre/pybroker#158] [Q&A] [answered]
Refering to this [Issue](https://github.com/edtechre/pybroker/issues/45#issuecomment-1622320598)

@edtechre @Pirat83 whats the current recommendation for backtesting with leverage?
Faking large enough initial cash?

I too would like to include some leverage in long and short positions

-`ctx.calc_target_shares(target)` is limited to 100% of cash.

-and if I do this:
`ctx.buy_shares = more shares than I have cash for`
it looks like it too doesnt work.

**Comment by edtechre** [comment, marked as answer]
Hi @SW4T400,

The library does not support leverage, as different brokers have varying margin requirements. Implementing these requirements is beyond the scope of the library. Instead, the library focuses on its primary use case: identifying effective trading signals independent of leverage. If you wish to simulate trading with higher capital, you can set a larger ```initial_cash``` parameter.
